Cargo.toml

```diff
@@ -26,20 +26,19 @@ jsonwebtoken = "~7.2"
 lazy_static = "~1.4"
 log = "~0.4"
 percent-encoding = "~2.1"
-protobuf = { version = "^2.0", features = [ "with-bytes" ] }
+protobuf = { version = "^2.20", features = [ "with-bytes" ] }
```

Comment:
This is needed because this project isn't following semver and 2.19 isn't compatible?

Reply:
Correct. @coltfred had a discussion with the project owner, who didn't seem to care that the bytes upgrades were not compatible. So that's a bit worrisome going forward, but he did say that he hopes bytes will be compatible from now on.

Reply:
https://github.com/stepancheg/rust-protobuf/issues/539